exporters/bypasses/stream_bypass.py

```diff
@@ -4,8 +4,9 @@
 
 from exporters.bypasses.base import BaseBypass
 from exporters.module_loader import ModuleLoader
+from exporters.iterio import cohere_stream
 
-Stream = namedtuple('Stream', 'file_obj filename size')
+Stream = namedtuple('Stream', 'filename size meta')
```

Comment:
Never, none of the readers we have use it but it's there for any
serializable reader-specific metadata it might need.

On Aug 9, 2016 22:50, "Elias Dorneles" <notifications@github.com> wrote:

> In exporters/bypasses/stream_bypass.py
> <https://github.com/scrapinghub/exporters/pull/326#discussion_r74131708>:
>
> >
> > -Stream = namedtuple('Stream', 'file_obj filename size')
> > +Stream = namedtuple('Stream', 'filename size meta')
>
> I don't get it, it seems that meta is empty in all Stream initializations.
> When is meta actually set?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/scrapinghub/exporters/pull/326/files/d08cb67108f9873369bf3e7a9dfcd21d43b793c7#r74131708>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AANrvcpbSchQ7c5DL4J3f0Hcrb3GddRLks5qeNoRgaJpZM4JdbR->
> .
>